### PR TITLE
(WIP) External wallet: showing networks that user disabled

### PR DESCRIPTION
**Pull request by NotEternal**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [ ] Good naming (as clear and simple as possible)
- [ ] Correct behavior if external API endpoints are down, return 404, 504 (or no answer), 401 errors (ddos simulation)
- [ ] I tested desktop/mobile resolution
- [ ] I tested light/dark theme
- [ ] I tested different languages
- [ ] I checked the functionality once again (**AFFECT MONEY**)
- [x] I checked the work on the Testnet
- [ ] I checked the work on the Mainnet
- [ ] I checked the work in the plugin
- [x] I checked the **PR** once again

## Tests

Please start auto tests as follows:

- add a label <ins>swap test</ins> to start swap tests
- add a label <ins>withdraw test</ins> to start withdraw tests

You can skip these tests if you completely sure that your changes aren't related to this functional

### Original issue
#4817
#4819

### CODE REVIEW

**Comment by developer24-exe** [comment]
builded https://testnet4820.swaponline.site ;

File: src/front/shared/components/modals/ConnectWalletModal/ConnectWalletModal.tsx [hunk 1]
```diff
@@ -21,9 +21,16 @@ class ConnectWalletModal extends React.Component<any, any> {
   constructor(props) {
     super(props)
 
+    const availableNetworks = Object.values(externalConfig.evmNetworks).filter(
```
Comment (VitaliyShulik) [inline src/front/shared/components/modals/ConnectWalletModal/ConnectWalletModal.tsx:24]: может эту логику добавить в `externalConfig` и инициализировать `evmNetworks` and `evmNetworkVersions`, а то это как костыль выглядет, или же перенести логику в `helpers/metamask`

Comment (noxonsu) [reply src/front/shared/components/modals/ConnectWalletModal/ConnectWalletModal.tsx:24]: > или же перенести логику в helpers/metamask,

а если клиент траст валет?

Comment (NotEternal) [reply src/front/shared/components/modals/ConnectWalletModal/ConnectWalletModal.tsx:24]: в них нет переключения внутреннего. Какую сеть мы укажем при подключении, такая и будет с кошелька использоваться

**Comment by developer24-exe** [comment]
commit 9fc42198c rebuilded https://testnet4820.swaponline.site https://mainnet4820.swaponline.site

**Comment by developer24-exe** [comment]
commit f7b0c4f52 rebuilded https://testnet4820.swaponline.site https://mainnet4820.swaponline.site

**Pull request closed without merging**